db/schema.rb

```diff
@@ -20,6 +20,17 @@
     t.index ["jti"], name: "index_jwt_blacklist_on_jti"
   end
 
+  create_table "recipes", force: :cascade do |t|
+    t.string "title", default: "", null: false
+    t.string "description", default: "", null: false
+    t.text "ingredients", default: [], array: true
+    t.text "directions", default: [], array: true
+    t.string "difficulty", default: "Intermediate", null: false
+    t.integer "duration", default: 0, null: false
+    t.integer "servings", default: 2, null: false
+    t.string "photos", default: [], array: true
+  end
+
```

Comment:
Looks like you need another `db:migrate` to do tags.

I think we'll also want to be able to join recipes and tags, so we'll need a join table and a many-to-many relationship. Might be worth checking out the intransitive association part of [this doc](https://www.sitepoint.com/master-many-to-many-associations-with-activerecord/).